app/controllers/search_controller.rb

```diff
@@ -0,0 +1,16 @@
+class SearchController < ApplicationController
+  def search
+    
+  end
+
+  def suggest
+    if params['query'].to_s != ''
```

Comment:
A params az egy olyan hash amiben a kulcsokat stringkent es symbolkent is el lehet erni. A rails konvencio a symbol hasznalata. Szoval `params[:query]` lehetne itt.

A masik hogy igy ellenorizni egy kulcs letezeset egy hashben elegge nyakatekert. Valamint az `else` ag nincs lekezelve es a rails az `app/views/search/suggest.js.erb`-t probalna meg renderleni ami nem letezik es elhalna a request. js es nem html mert ez egy ajax request.

Valami hasonlo elegansabb lenne:

```ruby
if params.key?(:query) # vagy eppen params.include?(:query)
  @results = SearchQuery.new(params[:query))
else
  @results = []
end
```